x-pack/solutions/observability/plugins/observability/public/application/index.tsx

```diff
@@ -28,7 +28,7 @@ import { HideableReactQueryDevTools } from './hideable_react_query_dev_tools';
 function App() {
   return (
     <>
-      <Routes>
+      <Routes enableExecutionContextTracking={true}>
```

Comment:
Sure, I started by enabling this logic everywhere in this [PR](https://github.com/elastic/kibana/pull/201118), assuming any custom implementation will override it. But when I asked teams to verify this assumption, in some cases, that was not the case, and when Anton and I were checking the issue, it seemed there could be a race condition, and we cannot rely on the order of execution here. 

So, I decided to disable this logic where we have custom implementation and have it out of the box everywhere else, but due to the following reasons, I decided to change direction:
1. For security, there are many places where we need to disable default tracking, which didn't seem to be worth it.
2. We cannot ensure in places that we enable it, it works as expected. So, allowing each team to enable it to verify the changes seemed like a better approach.
3. PR review in case of disabling would take a lot of time since different teams need to check the existing implementation for all their routes in one PR.